src/OmniSharp.DotNet/Models/DotNetProjectInformation.cs

```diff
@@ -9,12 +9,16 @@ public DotNetProjectInformation(ProjectContext projectContext, string configurat
         {
             this.Path = projectContext.RootProject.Path;
             this.Name = projectContext.ProjectFile.Name;
+            this.TargetFramework = new DotNetFramework(projectContext.TargetFramework);
 
             var outputPaths = projectContext.GetOutputPaths(configuration);
             this.CompilationOutputPath = outputPaths.CompilationOutputPath;
             this.CompilationOutputAssemblyFile = outputPaths.CompilationFiles.Assembly;
             this.CompilationOutputPdbFile = outputPaths.CompilationFiles.PdbPath;
 
+            var compilationOptions = projectContext.ProjectFile.GetCompilerOptions(targetFramework: projectContext.TargetFramework, configurationName: configuration);
```

Comment:
maybe this extension method would be better https://github.com/dotnet/cli/blob/7c6a2ad8b359e94ea35cf1fbe69b2d24da43bd34/src/Microsoft.DotNet.Compiler.Common/ProjectContextExtensions.cs#L20?